test/build_and_test.sh

```diff
@@ -2,6 +2,11 @@
 
 set -euo pipefail
 
+if [[ "$TRAVIS_EVENT_TYPE" -e "cron" ]]
```

Comment:
Just wondering where we can set Travis event type?

Reply:
You can create a cron build in Travis like this, in which case the env. variable TRAVIS_EVENT_TYPE is set to value "cron", link to docs: https://docs.travis-ci.com/user/cron-jobs/ 
![image](https://user-images.githubusercontent.com/1167948/38484208-75945078-3bf3-11e8-999f-439f800b4839.png)

Reply:
Oh! Cool!